### PR TITLE
Bootstrap chaindata download

### PR DESCRIPTION
**Pull request by ghost**
implementation of chain.acc data as per disccusion in #55 
* requires /client/chain-hash.json to exist on https://neo.org - see https://github.com/neo-project/neo.org/pull/4 (these are simply md5 hash of the two chain files)
* bootstrap configuration (filenames, hash file location and mirrors) are in config.(mainnet|testnet).json

i've setup an s3 instance as the first mirror but it is only on a free tier so it would be great to get a few more of these setup. they should only contain chain.acc.zip and chain.acc.test.zip and the hash file should always be on neo.org

### CODE REVIEW

**Comment by Ray-77** [comment]
The download process of client got stuck in 446MB  without throwing an exception.
59error
And we also need to surf  the internet appropriately to download the neo-chain-mirror. It got stuck in 326 MB , too. 
59error1

**Comment by ghost** [comment]
HI Ray, Thanks for testing - I'll look at whether or not webclient supports adding an exception for download timeout. It certainly doesn't support resume operations and I didn't want to be adding additional download libraries to the neo client. Could you please elaborate on what you mean by "And we also need to surf the internet appropriately to download the neo-chain-mirror" - I do not understand.

**Comment by Ray-77** [comment]
That means we need to climb over the "Great Firewall of China".

**Comment by ghost** [comment]
I see - that mirror is simply an amazon s3 bucket I setup for testing purposes. The mirror list is included in the neo configuration files. I'm in Australia so download from neo.org was very very slow for me. I would leave mirror list decisions to neo team.

File: neo-gui/config.json [hunk 1]
```diff
@@ -11,6 +11,14 @@
       "AddressUrl": "https://www.antchain.xyz/address/{0}",
       "AssetUrl": "https://www.antchain.xyz/asset/hash/{0}",
       "TransactionUrl": "https://www.antchain.xyz/tx/hash/{0}"
+    },
+    "BootstrapDownload": {
+      "file": "chain.acc.zip",
+      "hashList": "https://www.neo.org/client/chain-hash.json",
+      "mirrors": [
+        "https://neo.org/client",
+        "https://s3-ap-southeast-2.amazonaws.com/neo-chain-mirror"
```
Comment (shargon) [inline neo-gui/config.json:20]: Is this url is managed by neo council or city of zion council?

Comment (ghost) [reply neo-gui/config.json:20]: No - this is my personal AWS account and the bootstrap hasn't been updated since this PR was made

Comment (shargon) [reply neo-gui/config.json:20]: I think that should be (by default) `http://static.neo.org/client/chain.acc.zip`

**Comment by acunamario1988** [comment]
_£ /￦\€
[0x7f4d55f7fbc3a.txt.pdf](https://github.com/neo-project/neo-gui/files/2296545/0x7f4d55f7fbc3a.txt.pdf)
_

**Pull request closed without merging**